modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -209,20 +215,20 @@ class CondaCache {
      * @return the conda environment prefix {@link Path}
      */
     @PackageScope
-    Path createLocalCondaEnv(String condaEnv) {
+    Path createLocalCondaEnv(String condaEnv, String binaryName = "conda") {
         final prefixPath = condaPrefixPath(condaEnv)
         if( prefixPath.isDirectory() ) {
-            log.debug "Conda found local env for environment=$condaEnv; path=$prefixPath"
+            log.debug "${binaryName.capitalize()} found local env for environment=$condaEnv; path=$prefixPath"
```

Comment:
Since it's a binary name don't think should be capitalised, even better to between quotes. 

```
The binary '$name' was found local env for environment=$condaEnv; path=$prefixPath"
```

Reply:
Makes sense, I've updated the message here https://github.com/nextflow-io/nextflow/pull/2030/commits/976f15cd931a51a8fc61bdd8334e5198cde0d632